src/plugins/visualizations/public/index.ts

```diff
@@ -74,6 +75,7 @@ export type {
   HistogramParams,
 } from '../common/expression_functions/xy_dimension';
 export { urlFor, getFullPath } from './utils/saved_visualize_utils';
+export { convertSavedObjectAttributesToReferences } from './utils/saved_visualization_references';
```

Comment:
Why is convertSavedObjectAttributesToReferences getting exported? I could not find where this is used outside of this plugin.

Reply:
Left over from visualize refactor. Removing.